packages/excalidraw/element/textWysiwyg.tsx

```diff
@@ -224,6 +225,24 @@ export const textWysiwyg = ({
       }
 
       if (!container) {
+        if (element.autoResize === false) {
+          text = wrapText(
+            text,
+            getFontString(element),
+            Math.abs(updatedTextElement.width),
+          );
+        }
+
+        const metrics = measureText(
+          text,
+          getFontString(updatedTextElement),
+          updatedTextElement.lineHeight,
+        );
+
+        textElementWidth = updatedTextElement.width;
+        textElementHeight = metrics.height;
+
+        maxHeight = metrics.height;
```

Comment:
similarly to my comment about `refreshTextDimensions`, the fact you disable it there, you now need to handle part of that logic in here, and also start updating `height` & `text` of the text element below, while for `autoResize` text elements we do it upstream.

This leads to having both tight and lose coupling at the same time based on a property. Very hard to follow. Prone to errors I bet as well.